### PR TITLE
feat: text change

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

Changes the text when no open orders. Addresses: https://github.com/cowprotocol/cowswap/issues/3010

Screenshot 2023-08-07 at 17 07 01

### CODE REVIEW

File: src/modules/ordersTable/pure/OrdersTableContainer/index.tsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ export function OrdersTableContainer({
             ) : (
               <Trans>
                 You don&apos;t have any {isOpenOrdersTab ? 'open' : ''} orders at the moment. <br />
-                Create one for free! {/* TODO: add link for Advanced orders also */}
+                Time to create a new one! {/* TODO: add link for Advanced orders also */}
```
Comment (alfetopito) [inline src/modules/ordersTable/pure/OrdersTableContainer/index.tsx:213]: is the text changing for both order types?
Fine if it's, just checking

**Review by alfetopito — APPROVED** [review]
Approved with one question

File: src/modules/ordersTable/pure/OrdersTableContainer/index.tsx (hunk 1 above)
Comment (fairlighteth) [reply src/modules/ordersTable/pure/OrdersTableContainer/index.tsx:213]: Yes...

**Review by shoom3301 — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
Great, thank you!

**Pull request merged**